### ISSUE TITLE
CHAPTER1 #import(someLib) has changed to import 'someLib'

### ISSUE THREAD

**Issue by webdesign-benny** [dart4hipsters/dart4hipsters.github.com#9]
Hi,
Started yesterday with this nice tutorial.
However, I noticed that in Chapter1 
# import(someLib) is wrong and need to be

import 'someLib' instead.
I hope an update for this book is comming soon.
Cheers
Benny

**Comment by eee-c** [comment]
Yup, I'm working through the book now to update for M1+. I wish those guys would slow down!

In all seriousness, there are a bunch of little changes like that, plus I really need to completely rewrite the testing chapter. I'd also like to add a server side chapter. I still need to discuss with the Prags how all of that gets pushed out.

FWIW I have the master branch of [dart-comics](https://github.com/eee-c/dart-comics) updated for M1, but I still need to go back to get the your_first_dart_app branch updated.

Also, you've got the Prag edition (not the self published version), right?

**Comment by webdesign-benny** [comment]
Hi Chris,
Thx for your mail.
I bought the book last Wednesday from pragprog.com
And yup it's a Prag edition.
I would be delight to see the updated version dow !
Would you be so kind to inform me when the updated version is ready?
Thanks very much !
Benny

info
<deleted by Chris>

Op 30/11/2012 23:20, Chris Strom schreef:

**Comment by eee-c** [comment]
We only re-opened the source for updates yesterday. Still, I'm about 1/2 way through the [errata](http://pragprog.com/titles/csdart/errata). I expect to finish that list this weekend (and replace all of the `#import`s with just `import`). Then it's up to the Prags on when to release the next edition. When I self-publish, I can release updates right away, but they have this pesky need for “quality control” :P

I'm going to leave this GitHub issue open to remind myself to let you know as soon as the update is available. If you find any other issues, let me know!

**Comment by webdesign-benny** [comment]
Thx you very much :-)
happy weekend &
cheers
Benny

Op 1/12/2012 16:27, Chris Strom schreef:

**Comment by eee-c** [comment]
Wow. There were a _lot_ more changes than I realized, but the next release should be imminent. Lots of fixes and several new sections. Sorry it took this long, but hopefully the update will be made available in the next week or so.
